custom_components/bhyve/switch.py

```diff
@@ -275,6 +280,30 @@ async def async_turn_on(self, **kwargs: Any) -> None:
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn the switch off."""
         await self._set_state(False)
+        
+    async def start_program(self):
+        """Begins running a program."""
+        station_payload = self._program["program"]
+        await self._send_program_message(station_payload)
+
+    async def _send_program_message(self, station_payload):
+        try:
+            now = datetime.datetime.now()
+            iso_time = now.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+            payload = {
+                "event": EVENT_CHANGE_MODE,
+                "mode": "manual",
```

Comment:
It looks like these were copied from `_send_station_message`. Are these attributes required on this event? They smell funny to me... why would a program have a mode?

Reply:
They seem to be required, take a look at my findings that I pasted in issue #145 . I monitored the WS and that's what came across when running a program.